application/aspnet/sdk-example/example/example.csproj

```
@@ -0,0 +1,11 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="SecretHub" Version="0.1.0" />
```

Let's use the latest version here:
```suggestion
    <PackageReference Include="SecretHub" Version="0.2.0" />
```